.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/app.log
```

Comment:
The line `/logs/app.log` is intended to ignore the `app.log` file in the `logs` directory. However, the leading slash suggests that it is an absolute path from the root of the repository. If you want to ignore all `app.log` files within any `logs` directory, you should use `logs/app.log` without the leading slash.